src/components/views/ScreeningsCard.vue

```diff
@@ -0,0 +1,121 @@
+<script>
+import { defineComponent } from 'vue';
+import MoviePoster from '@/components/common/Movie/MoviePoster.vue';
+import MovieLength from '@/components/common/Movie/MovieLength.vue';
+import AppButton from '@/components/common/App/AppButton.vue';
+import MovieTitle from '@/components/common/Movie/MovieTitle.vue';
+import AppTag from '@/components/common/App/AppTag.vue';
+import { getScreeningsByMovieAndDate } from '@/api/service/Screenings';
+
+export default defineComponent({
+  components: { MoviePoster, MovieLength, AppButton, MovieTitle, AppTag },
+  props : {
+    movieData: {
+      type: Object,
+      required: true,
+    },
+    daySelected: {
+      type: Date,
+      default: new Date().toISOString().slice(0, 10)
+    },
+    titleSelected: {
+      type: String,
+      default: 'All movies',
+    }
+  },
+  data() {
+    return {
+      screenings: [],
+      selectedMovieId: this.movieData.id,
+    }
+  },
+  watch: {
+    daySelected(newDay, previousDay) {
+      if (newDay !== previousDay) {
+        this.getScreenings()
+      }
+    },
+    titleSelected(newOption, previousOption) {
+      if (newOption !== previousOption) {
+        this.selectedMovieId = this.getMovieIdByTitle(this.movieData, newOption)
+        this.getScreenings()
+      }
+      if (newOption === 'All movies') {
+        this.selectedMovieId = this.movieData.id
+        this.getScreenings()
+      }
+    }
+  },
+  methods: {
+    getScreeningTime(screening) {
+      return screening.datetime.substring(11, 16)
+    },
+    async getScreenings() {
+      try {
+        const response = await getScreeningsByMovieAndDate(this.selectedMovieId, this.daySelected);
```

Comment:
Just to let you know: both search params (`movie_id` and `date`) that you send to the `/seances` endpoint are optional. 

So, if the user doesn't select any movie, you can skip the `movie_id` param: (/seances?~movie_id=123~date=2022-08-30) and you will get all seances for the specific day.

Reply:
Good point! But at this moment it seems simpler to me to use both params as I generate ScreeningsCards per movie. But will re-think it tomorrow, when my grey cells regenerate a bit 😉